src/main/java/competition/subsystems/internalconveyor/commands/ManualLiftCommand.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.internalconveyor.commands;
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.internalconveyor.KickerSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ManualLiftCommand extends BaseCommand{
```

Comment:
With this name it's not exactly clear what's being lifted. I suggest `KickerLiftCommand`